### DISCUSSION TITLE
feature request: an estimate of full sync data size

### DISCUSSION THREAD

**Discussion by anipenguin** [nervosnetwork/neuron#2647] [Ideas]
Request: Providing an estimate of the fully synced node's size or displaying this information somewhere before syncing.

Hey, it would be great if there was some way to see how much space the fully synced node would take up before starting the sync. A lot of users have been asking about this and it would be super helpful to have that info upfront!

**Comment by Keith-CY** [comment]
The size of a fully synced node grows with time, so it cannot be delivered within Neuron.

We're building a website for Neuron and the metrics of neuron, as follows:

1. tip block number
2. block size
3. assumed valid target block number

can be listed there @Danie0918 @Sven-TBD

**Reply by anipenguin** [reply]
I see, as long as users can find this info 😇

**Reply by Keith-CY** [reply]
This feature is tracked by https://github.com/Magickbase/neuron-public-issues/issues/255
